src/Symfony/Component/Config/Exception/FileLoaderLoadException.php

```diff
@@ -34,9 +34,20 @@ public function __construct($resource, $sourceResource = null, $code = null, $pr
 
         // Is the resource located inside a bundle?
         if ('@' === $resource[0]) {
-            $parts = explode(DIRECTORY_SEPARATOR, $resource);
-            $bundle = substr($parts[0], 1);
-            $message .= ' '.sprintf('Make sure the "%s" bundle is correctly registered and loaded in the application kernel class.', $bundle);
+            $bundleController = substr($resource, 1);
+            $bundle = strstr($bundleController, '/', true);
+
+            // Get the controller full path
+            $currentPath = dirname(__FILE__);
+            $srcPath = strstr($currentPath, 'vendor/', true);
```

Comment:
is this correct? i know composer vendor can be also configured to be different directory no?